app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+
+  USER_CATEGORY = [
+    ["manager", "Manager"],
+    ["developer", "Developer"],
+    ["qa", "QA"],
+  ]
```

Comment:
We can use rails method `.titalize` to convert "manager" to "Manager".
If we still need this then move this into helper instead of model

Reply:
I am moving this to helper